### DISCUSSION TITLE
wow, just what I was looking for.

### DISCUSSION THREAD

**Discussion by nelg** [taligentx/dscKeybusInterface#46] [dscKeybusInterface Discussion]
Not an issue, just wanted to say great work!.

I've been running an arduino connected to my DSC for about 6 years, (http://dsc.juliano.com.br/)
What you are doing with the ESP8266 is just what I was looking for.   Really excited to see this work, I'll be giving the stuff you've written a go soon.

Thanks
Glen Ogilvie

**Comment by zzhang1** [comment]
Sorry for adding noise to the issues tracker, but I just wanted to say thanks as well. This project helped me breathe life into an ancient 32-zone DSC PC5010 system. If you hadn't painstakingly reverse-engineered the keybus interface, I would have ended up having to tap each sensor individually and still be stuck with read-only access. Thank you for sharing this amazing work!

**Reply by jambuur** [reply]
I want to say a big thank you as well! We moved house recently, the PC1616-based alarm had been decommissioned a long time ago with no codes available. The monitoring company on the sticker couldn't help either as they lost all their legacy DSC info due to a computer crash. The Unlocker script did the trick, so cool.

**Comment by taligentx** [comment]
@nelg @zzhang1 - Thank you!  It's great to hear your experiences.

**Comment by tylerbinski** [comment]
Thank you as well ! this is awesome and the amount of examples is impressive!
